test/test_sparse_csr.py

```diff
@@ -0,0 +1,202 @@
+import torch
+
+torch.set_default_dtype(torch.double)
```

Comment:
please don't do this. I doubt your tests actually need it and we're trying to get rid of all of these.